.devops/deploy-pipelines.yml

```diff
@@ -171,6 +171,7 @@ stages:
               REACT_APP_URL_API_PARTY_REGISTRY_PROXY: '$(react_app_url_api_party_registry_proxy)'
               REACT_APP_ENABLE_ASSISTANCE: '$(react_app_enable_assistance)'
               REACT_APP_PAGOPA_HELP_EMAIL: '$(react_app_pagopa_help_email)'
+              REACT_APP_URL_INSTITUTION_LOGO_PREFIX: '$(react_app_url_storage)/institutions/'

```

Comment:
check the dashboard deploy pipeline, and copy also the population of the react_app_url_storage variable